src/sentry/conf/server.py

```diff
@@ -3294,12 +3296,15 @@ def build_cdc_postgres_init_db_volume(settings: Any) -> dict[str, dict[str, str]
     KAFKA_INGEST_TRANSACTIONS: {"cluster": "default"},
     # Topic for receiving metrics from Relay
     KAFKA_INGEST_METRICS: {"cluster": "default"},
+    # Topic for routing invalid messages from KAFKA_INGEST_METRICS
+    KAFKA_INGEST_METRICS_DLQ: {"cluster": "default"},
     # Topic for indexer translated metrics
     KAFKA_SNUBA_METRICS: {"cluster": "default"},
     # Topic for receiving profiles from Relay
     KAFKA_PROFILES: {"cluster": "default"},
     KAFKA_INGEST_PERFORMANCE_METRICS: {"cluster": "default"},
     KAFKA_SNUBA_GENERIC_METRICS: {"cluster": "default"},
+    KAFKA_INGEST_PERFORMANCE_METRICS_DLQ: {"cluster": "default"},
```

Comment:
Isn't this generic metrics now? Why still calling it performance?

Reply:
I'm assuming this is relic from the past, I am naming it `KAFKA_INGEST_PERFORMANCE_METRICS_DLQ` because the topic is is `KAFKA_INGEST_PERFORMANCE_METRICS `. If we want to change it, we should change both

Reply:
Yes we should. But at least use the new name now so there's only 1 rename needed not 2.

Reply:
I'm hesitant to do it in this pr because ops may have custom settings for different topics in other environments

Reply:
Alright, it's changed